src/pages/space/space.ts

```diff
@@ -16,7 +16,7 @@ export class SpacePage {
     params: NavParams
   ) {
     this.uid = params.get('uid');
-    this.spaceLink = `betwixt://betwixt.com/?group_uid=${this.uid}`;
+    this.spaceLink = `betwixt://betwixt.space/?group_uid=${this.uid}`;
```

Comment:
Are you sure there aren't any other changes you need to make?
I think @Palacios95 had mentioned that there was something else that needed to be changed...

Reply:
I believe that the "extra" thing we needed was for iOS, but that was a while back and I think it's resolved already.